src/annotator/integrations/pdf.js

```diff
@@ -319,8 +319,17 @@ export class PDFIntegration {
     const maximumWidthToFit = window.innerWidth - sidebarLayout.width;
     const active = sidebarLayout.expanded && maximumWidthToFit >= MIN_PDF_WIDTH;
 
-    this.pdfContainer.style.width = active ? maximumWidthToFit + 'px' : 'auto';
-    this.pdfContainer.classList.toggle('hypothesis-side-by-side', active);
+    // If the sidebar is closed we reserve enough space for the toolbar controls
+    // so that they don't overlap a) the chevron-menu on the right side of
+    // PDF.js's top toolbar and b) the document's scrollbar.
+    //
+    // If the sidebar is open we reserve space for the whole sidebar if there is
+    // room, otherwise we reserve the same space as in the closed state to
+    // prevent the PDF content shifting when opening and closing the sidebar.
```

Comment:
Pardon the pedant here, but I think a couple of additional commas would aid in readability:

```suggestion
    // If the sidebar is closed, we reserve enough space for the toolbar controls
    // so that they don't overlap a) the chevron-menu on the right side of
    // PDF.js's top toolbar and b) the document's scrollbar.
    //
    // If the sidebar is open, we reserve space for the whole sidebar if there is
    // room, otherwise we reserve the same space as in the closed state to
    // prevent the PDF content shifting when opening and closing the sidebar.
```